src/inverseRobot.js

```diff
@@ -3,11 +3,25 @@
 /*
  * @param {object} robot
  *
- * @return {object}
+ * @return {object | null}
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixed = {};
```

Comment:
I would change the name of this constant so it's more descriptive. Maybe `fixedRobot`?